tests/lib/rules/no-restricted-globals.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * @fileoverview Tests for no-restricted-globals.
+ * @author Benoît Zugmeyer
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var rule = require("../../../lib/rules/no-restricted-globals"),
+    RuleTester = require("../../../lib/testers/rule-tester");
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+var ruleTester = new RuleTester();
+ruleTester.run("no-restricted-globals", rule, {
+    valid: [
+        { code: "foo" },
+        { code: "foo", options: ["bar"] },
+        { code: "var foo = 1;", options: ["foo"] },
+        { code: "event", env: { browser: true }, options: ["bar"] },
+        { code: "import foo from 'bar';", options: ["foo"], parserOptions: { ecmaVersion: 6, sourceType: "module" } },
+        { code: "function foo() {}", options: ["foo"] },
+        { code: "function fn() { var foo; }", options: ["foo"] }
```

Comment:
I admit this is a pretty obvious case and not directly related to the rule, but maybe we could use a negative valid test on restricted variables being properties? (E.g., `foo.bar` should be valid when "bar" is specified.)